ui/packages/catalog-ui-search/src/main/webapp/component/search-interactions/search-interactions.view.js

```diff
@@ -108,6 +110,26 @@ module.exports = Marionette.LayoutView.extend({
     user.savePreferences()
     this.triggerCloseDropdown()
   },
+  triggerTypeBasic: function() {
+    let oldType = this.model.get('type')
+    if (oldType === 'custom' || oldType === 'new-form') {
+      this.model.set('title', 'Search Name')
```

Comment:
I don't think we need to reset the title, it's entirely possibly the user has changed it and we don't want to throw that change away.

Reply:
Removed